core/drivers/crypto/crypto_api/include/drvcrypt_acipher.h

```diff
@@ -133,6 +133,17 @@ struct drvcrypt_secret_data {
 	struct drvcrypt_buf secret; /* Shared secret */
 };
 
+/*
+ * ECC Encrypt data
+ */
+struct drvcrypt_encrypt_data {
+	uint32_t algo;
```

Comment:
please describe each field as done above

maybe rename `drvcrypt_encrypt_data` to `drvcrypt_cipher_data` (enc & dec)